include/constants/battle_config.h

```diff
@@ -106,6 +106,7 @@
 #define B_MOODY_ACC_EVASION         GEN_6 // In Gen8+, Moody CANNOT raise Accuray and Evasion any more.
 #define B_FLASH_FIRE_FROZEN         GEN_6 // In Gen5+, Flash Fire can trigger even when frozen, when it couldn't before.
 #define B_SYNCHRONIZE_NATURE        GEN_6 // In Gen8+, if the Pokémon with Synchronize is leading the party, it's 100% guaranteed that wild Pokémon will have the same ability, as opposed to 50% previously.
+#define B_EXPANDED_INTIMIDATE       GEN_8 // In Gen8, Intimidate doesn't work on opponents with the Inner Focus, Scrappy, Own Tempo or Oblivious abilities.
```

Comment:
i'm not sure if `B_EXPANDED_INTIMIDATE` is the best name given that what happened to intimidate was the opposite of an expansion.

would something like `B_INTIMIDATE_BLOCK` be better maybe, or is it fine as is?

Reply:
`B_INTIMIDATE_BLOCK` would be more appropriate if Intimidate didn't already block certain abilities by default _(and it does, more specifically Hyper Cutter, Clear Body and White Smoke)_.

My reasoning behind `B_EXPANDED_INTIMIDATE` is that here, I basically expanded the ability's reach in terms of which opposing abilities can block it from activating.
I'm opened to suggestions if that's no good though.

Reply:
As discussed on Discord, I like Lunos's suggestion of `B_UPDATED_INTIMIDATE` most.

Reply:
Yeah, me too. I think he got a good idea this time around. I'll probably stick to it. His label reflects the effect of the configuration better.